### PR TITLE
Import sheets 2 & 3

### PR DESCRIPTION
**Pull request by ascii-dev**
## Description
Imports sheets 2 & 3 of the 2015 report file.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: reports/historical/_gmmp_2015_importer.py [hunk 1]
```diff
@@ -47,6 +47,57 @@ def import_1(self, sheet_info):
 
         return all_data
 
+    def import_2(self, sheet_info):
+        all_data = dict()
+        for year, col_start, col_end in [
+            (2015, 3, 10),
+            (2010, 13, 16)
+        ]:
+            data = dict()
```
Comment (kilemensi) [inline reports/historical/_gmmp_2015_importer.py]:
```suggestion
            data = {}
```

File: reports/historical/_gmmp_2015_importer.py [hunk 2]
```diff
@@ -47,6 +47,57 @@ def import_1(self, sheet_info):
...
+                'Caribbean': (54, 68),
+                'Europe': (71, 100),
+                'Latin America': (103, 116),
+                'Middle East': (119, 124),
+                'North America': (127, 128),
+                'Pacific Island': (131, 134),
+            }
+            for continent in continents.keys():
+                row_start, row_end = continents[continent]
+                continental_data = dict()
```
Comment (kilemensi) [inline reports/historical/_gmmp_2015_importer.py]:
```suggestion
            for continent, (row_start, row_end) in continents.items():
                continental_data = {}
```

**Review by kilemensi — COMMENTED** [review]
:+1: @ascii-dev 

Make `historical_detail.py` consistent: If we start with most recent year, 2015 followed by historical year, 2010, then lets follow the same structure for all sheets. We kind of of mixing them now (`ws_01`, `ws_05` differ from `ws_02`, `ws_03`).

----------------------------

QQ: When a sheet like `ws_02` has continents (which I think should be named regions by the way since that's what GMMP uses in it's report), and country names, doesn't that mean that report should have reports for `['global', 'region', and 'country']`? Or is there another place where we can look up to see what kind of report we're expected to produce per sheet?

**Comment by ascii-dev** [comment]
> QQ: When a sheet like `ws_02` has continents (which I think should be named regions by the way since that's what GMMP uses in it's report), and country names, doesn't that mean that report should have reports for `['global', 'region', and 'country']`? Or is there another place where we can look up to see what kind of report we're expected to produce per sheet?

@kilemensi if I get you right, for the `import_2`, for instance, we should split the report imported by the method in respect to whether `global`, `region`, or `country`, is parsed along with the command, right?

File: reports/historical/_gmmp_2015_importer.py [hunk 3]
```diff
@@ -47,6 +47,56 @@ def import_1(self, sheet_info):
 
         return all_data
 
+    def import_2(self, sheet_info):
+        all_data = dict()
```
Comment (kilemensi) [inline reports/historical/_gmmp_2015_importer.py]:
```suggestion
        all_data = {}
```

**Review by kilemensi — APPROVED** [review]
:rocket:

**Pull request merged**